qiskit_machine_learning/algorithms/classifiers/qsvc.py

```diff
@@ -47,16 +48,23 @@ def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwar
             **kwargs: Arbitrary keyword arguments to pass to SVC constructor.
         """
 
+        if (len(args)) != 0:
+            msg = (
+                f"Positional arguments ({args}) are deprecated as of version 0.3.0 and "
+                f"will be removed no sooner than 3 months after the release. Instead use "
+                f"keyword arguments. Positional arguments are ignored and not passed to the superclass."
```

Comment:
`Positional arguments are ignored and not passed to the superclass.` This is not the case correct so should be deleted from the message